spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -255,7 +255,7 @@
 "GNU Lesser General Public License v3 (LGPLv3)": LGPL-3.0-only
 "GNU Lesser General Public License v3 or later (LGPLv3+)": LGPL-3.0-or-later
 "GNU Lesser General Public License v3+": LGPL-3.0-or-later
-"GNU Lesser General Public License": LGPL-2.1-only
+"GNU Lesser General Public License": LGPL-2.1-or-later
 "GNU Lesser General Public License, Version 2.1": LGPL-2.1-only
 "GNU Lesser Public License": LGPL-2.1-only
 "GNU Library or Lesser General Public License (LGPL)": LGPL-2.1-only
```

Comment:
Why did you not change these to  LGPL-2.1-or-later?

Reply:
Simply an oversight. I now when through all mapping to `LGPL-2.1-only` and found a bunch of more to fix.